docs/collections/_policies/syntax-grammar.md

```diff
@@ -151,7 +151,7 @@ Member ::= Primary {Access}
 ## `Annotation` {#grammar-annotation}
 
 ```
-Annotation ::= '@'ANYIDENT'('STR')'
+Annotation ::= '@'ANYIDENT' ( ('STR')' )?
```

Comment:
not sure normal conventions for this grammar syntax, but is this clear that the entire element is optional, and not just the final `)`?

Reply:
I think it's clear, but  I did have a `'` in the wrong place, so maybe it's more clear neow